webrtc.html

```diff
@@ -1526,83 +1526,99 @@
+                            of view.</p>
+                          </li>
+                          <li>
+                            <p>As described by <span data-jsep=
+                            "applying-a-remote-desc">[[!JSEP]]</span>, attempt to
+                            find an existing <code><a>RTCRtpTransceiver</a></code>
+                            object, <var>transceiver</var>, to represent the media
+                            description.</p>
+                          </li>
+                          <li>
+                            <p> If no suitable transceiver is found
+                            (<var>transceiver</var> is unset), run the following
+                            steps:</p>
+                            <ol>
+                              <li>
+                                <p><a>Create an RTCRtpSender</a>, <var>sender</var>,
+                                from the media description.</p>
+                              </li>
+                              <li>
+                                <p><a>Create an RTCRtpReceiver</a>,
+                                <var>receiver</var>, from the media description.</p>
+                              </li>
+                              <li>
+                                <p><a>Create an RTCRtpTransceiver</a> with
+                                <var>sender</var>, <var>receiver</var> and
+                                <var>direction</var> set to <code>"recvonly"</code>,
+                                and let <var>transceiver</var> be the result.</p>
+                              </li>
+                            </ol>
+                          </li>
+                          <li>
+                            <p>Set <var>transceiver</var>'s <code><a data-for=
+                            "RTCRtpTransceiver">mid</a></code> value to the mid of
+                            the corresponding media description. If the media
+                            description has no MID, and <var>transceiver</var>'s
+                            <code><a data-for="RTCRtpTransceiver">mid</a></code>
+                            is unset, generate a random value as
+                            described in <span data-jsep=
+                            "applying-a-remote-desc">[[!JSEP]]</span>.</p>
+                          </li>
+                          <li>
+                            <p>If <var>direction</var> is <code>sendrecv</code> or
+                            <code>recvonly</code>, and
+                            <var>transceiver</var>'s
+                            <a>[[\CurrentDirection]]</a> slot
+                            is neither "sendrecv" nor "recvonly",
+                            <a>process the addition of a remote track</a> for
+                            the media description, given <var>transceiver</var>
+                            and <var>trackEvents</var>.</p>
```

Comment:
That won't happen, because there's a check for whether `[[CurrentDirection]]` is `"sendrecv"` or "`recvonly`". Meaning the event will only fire if the remove peer is offering to send something, and nothing is currently being received.